guide/payments/receive.md

```diff
@@ -0,0 +1,213 @@
+   width = 250
+   height = 541
+%}
+
+{% include picture.html
+   image = "/assets/images/guide/payments/receive/enter-details-purpose.png"
+   retina = "/assets/images/guide/payments/receive/enter-details-purpose@2x.png"
+   alt-text = "Screen asking the receiver to enter the purpose of the payment"
+   caption = "Sometimes the purpose can be determined by identifying who the payment is from. Be mindful of that, and consider easy ways to help the receiver add more context."
+   width = 250
+   height = 541
+%}
+</div>
+
+If your design’s goal is to get the receiver in the quickest time to share an address, you can choose to show them a new address straight away. The other properties can be left as optional fields. Consider the trade-off when optimizing this way, as there will be less useful contextual details for reports that you may want to provide in the future.
+
+<div class="image-slide-gallery">
+
+{% include picture.html
+   image = "/assets/images/guide/payments/receive/enter-details-compact.png"
+   retina = "/assets/images/guide/payments/receive/enter-details-compact@2x.png"
+   alt-text = "Screen showing QR Code for the address and optional entry for other properties"
+   caption = "All properties editable on a single screen"
+   width = 250
+   height = 541
+%}
+
+</div>
+
+<!-- 
+Update interopability page
+Since there are many Bitcoin applications for people to choose from, its likely that the sender and receiver are not using the same one and they don't have the abilitiy to exchange payment details within the application itself. 
+
+The exchange of payment details then most commonly happens outside of the wallet application. This 
+
+Since bitcoin is a open system and has many payment applications built ontop of it, there is a [standard format](https://bitcoin.design/guide/foundations/wallet-interoperability/#payment-links) of "payment links" that most bitcoin applications use to share payment details.
+-->
+
+## Choosing the type of address
+There are currently [three different versions](https://bitcoin.design/guide/glossary/#address) of bitcoin addresses. In order of oldest to newest they are; Legacy, Script, and SegWit. It is best practice for new applications to support SegWit, the latest, by default.
+
+The most commonly used addresses are still the “legacy” addresses since Segwit addresses are only a few years old, and some older applications have yet to update.
+
+Suppose the sender is using a legacy wallet and paying to the receiver's SegWit address. In this case, the sender's wallet may incorrectly warn them that the address is invalid or not supported. This can confuse the sender’s end, leading them to think that the receiver provided an incorrect address.
+
+The receiver should then have the ability to switch to a Script address that does not have all the benefits of SegWit, like cheaper transactions but will be compatible with the sender's wallet.
```

Comment:
We should clearly give directions here, I don't think it's clear people on what to use.

Reply:
This is already in one of the screens. I created another one that alerts a user that the format of the address they entered is not supported. [Check it](https://www.figma.com/file/VB3GQdAnhl8yta44DY3PSV/Bitcoin-Wallet-UI-Kit?node-id=1366%3A32884).

Reply:
@GBKS I think that graphic would more be suited on the send page. Which can be updated to reflect this same issue from the sender's perspective.

Reply:
Sounds good. Feel free to use (and not use) those graphics as you think works best.

Reply:
thx @GBKS added it